src/server/graph-generation/interaction/index.js

```diff
@@ -0,0 +1,170 @@
+const _ = require('lodash');
+const fetch = require('node-fetch');
+const pc = require('../../pathway-commons');
+const geneValidator = require('../../enrichment/validation').validatorGconvert;
+const logger = require('./../../logger');
+
+function edgeType(type) {
+  switch(type){
+    case 'in-complex-with':
+    case 'interacts-with':
+      return 'Binding';
+    case 'controls-phosphorylation-of':
+      return 'Phosphorylation';
+    case 'controls-expression-of':
+      return 'Expression';
+    default:
+      return '';
+  }
+}
+
+function getInteractionInfoFromPC(sources) {
+  const idArray = _.uniq(_.concat([], sources)); //convert sources to array
```

Comment:
As mentioned above, let's strip out the anything that populates `metaData.networkMetadata`.